src/compiler/factory/emitHelpers.ts

```diff
@@ -34,6 +34,7 @@ namespace ts {
         // Class Fields Helpers
         createClassPrivateFieldGetHelper(receiver: Expression, state: Identifier, kind: PrivateIdentifierKind, f: Identifier | undefined): Expression;
         createClassPrivateFieldSetHelper(receiver: Expression, state: Identifier, value: Expression, kind: PrivateIdentifierKind, f: Identifier | undefined): Expression;
+        createClassPrivateFieldInHelper(receiver: Expression, state: Identifier): Expression;

```

Comment:
I recommend swapping the order of `receiver` and `state`, per my comments below on `classPrivateFieldInHelper`.